b2b-sample/guardio-insurance-sample-application/components/sections/sections/settingsSection/roleManagementSection/otherComponents/createRoleComponent/createRoleComponent.js

```diff
@@ -0,0 +1,146 @@
+/**
+ * Copyright (c) 2022, WSO2 LLC. (https://www.wso2.com). All Rights Reserved.
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import React, { useState } from "react";
+import { Loader, Modal, Panel, Steps, useToaster } from "rsuite";
+import General from "./createRoleComponentInner/general";
+import Permission from "./createRoleComponentInner/permission";
+import Users from "./createRoleComponentInner/users";
+import styles from "../../../../../../../styles/Settings.module.css";
+import { decodeCreateRole } from "../../../../../../../util/apiDecode/settings/role/deocdeCreateRole";
+import { LOADING_DISPLAY_BLOCK, LOADING_DISPLAY_NONE } from "../../../../../../../util/util/frontendUtil/frontendUtil";
+import { errorTypeDialog, successTypeDialog } from "../../../../../../common/dialog";
+
+export default function CreateRoleComponent(prop) {
+
+    const { open, setOpenCreateRoleModal, session } = prop;
+
+    const [ loadingDisplay, setLoadingDisplay ] = useState(LOADING_DISPLAY_NONE);
+    const [ step, setStep ] = useState(0);
+    const [ displayName, setDisplayName ] = useState("");
+    const [ permissions, setPermissions ] = useState([]);
+    const [ users, setUsers ] = useState([]);
+
+    const toaster = useToaster();
+
+    /**
+     * change the screens to previous and next
+     * 
+     * @param nextStep - next step
+     */
+    const onChange = (nextStep) => {
+        setStep(nextStep < 0 ? 0 : nextStep > 3 ? 3 : nextStep);
+    };
+    const onPrevious = () => onChange(step - 1);
+    const onNext = () => {
+        if (step == 2) {
+
```

Comment:
make it consistent through out the code to leave a new line after the if condition or not.

Reply:
Fixed with https://github.com/wso2/samples-is/pull/416/commits/e535216c72dd635ee40f158f18eab8bdfffa3585